src/twisted/conch/ssh/filetransfer.py

```diff
@@ -488,7 +491,7 @@ def _ebStatus(self, reason, requestId, msg=b"request failed"):
             code = reason.value.code
             message = networkString(reason.value.message)
         else:
-            log.err(reason)
+            self.log.failure('Unknown error handling', reason)
```

Comment:
Likewise, it would be good to include the `requestId`.

Reply:
I seemed to have forgotten to come up with a better error message here anyways. I think something like `"Request {requestId} failed with unknown error: {message}"` could work, where message is `nativeString(message)`.